caliban/util.py

```diff
@@ -397,11 +397,15 @@ def capture_stdout(cmd: List[str],
   - return code of the process
 
   """
+  if file is None:
+    file = sys.stdout
```

Comment:
this was the default before inside of `print`, but I'd like to make ti more explicit here.

Reply:
Agree, being explicit when it doesn't add significant code is always good.